### PR TITLE
Fix End of JSON Input Integration Test Error

### LINKED ISSUE #417: webhook integration tests fail with auto bump of zipkin and candlelight
Below error happens starting at commit 1bcc2b92f299ea55fcf773d275dc65107a6b87c8 during local integration tests.  If that commit is reverted, the issue does not happen, even on the latest version.  

It might still be worth testing in CD to verify the error occurs there, too.  

The error happens for all the webhook tests in talaria/Connect_event_msgtype.robot (see image below)
body={"message":"failed to JSON unmarshal webhook: unexpected end of JSON input"}

image

### PR DESCRIPTION
**Pull request by renaz6**
- Fixes #417 
- https://github.com/xmidt-org/candlelight/pull/141 should be merged first
- Fix `end of JSON input` error by updating to new Candlelight version, which only decodes a request in the middleware if the request can be decoded into a WRP object. This avoids Tr1d1um trying to decode the payload a second time and running into issues.

### CODE REVIEW

**Review by denopink — APPROVED** [review]
there looks like to be a go.mod/go.sum conflict that needs to be resolve, other than that lgtm! 🍻

**Comment by renaz6** [comment]
> there looks like to be a go.mod/go.sum conflict that needs to be resolve, other than that lgtm! 🍻

Yeah! Waiting until the Candlelight PR is merged before updating!

**Pull request merged**
